### ISSUE TITLE
Show the namespace for objects when they only differ by that

### ISSUE THREAD

**Issue by jsoref** [argoproj/argo-cd#4248]
# Summary

We have two objects of the same type (e.g. Certificate) in distinct namespaces (e.g. `test` and `prod`) owned by a common application. In the graph, they appear roughly adjacent and it's hard to know which is which, it'd be nice if the namespace were shown in this case.

# Motivation

Make it easier to recognize which object is which w/o requiring clicking to disambiguate.

# Proposal

Stick the namespace on the icon line after icons.

**Comment by andrii-korotkov-verkada** [comment]
Yeah, might want to show the namespace. Although my 2 cents is that test and prod should be managed my separate apps, perhaps a part of replicaset.
